app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -57,8 +57,10 @@ def rrd_pdf_uploaded_to_s3?
     read_metadata(:pdf_guid).present?
   end
 
-  Uploader = RapidReadyForDecision::FastTrackPdfUploadManager
-  PDF_FILENAME_REGEX = /#{Uploader::DOCUMENT_NAME_PREFIX}.*#{Uploader::DOCUMENT_NAME_SUFFIX}/
+  DOCUMENT_NAME_PREFIX = 'VAMC'
+  DOCUMENT_NAME_SUFFIX = 'Rapid_Decision_Evidence'
+  PDF_FILENAME_REGEX = /#{DOCUMENT_NAME_PREFIX}.*#{DOCUMENT_NAME_SUFFIX}/
+  RRD_CODE = 'RRD'
```

Comment:
Not a big deal, but is there a reason these are defined here vs the beginning of the module?

Reply:
Not really a reason except that these constants are only used in this one place. And that I was going to remove them too and a lot of surrounding code, because I think it isn't being used, except that it's a little more unclear if it's used or not, so I wanted that to be a separate PR